packages/component-library-react/src/CardListItem.tsx

```diff
@@ -0,0 +1,97 @@
+/**
+ * @license EUPL-1.2
+ * Copyright (c) 2021 Robbert Broersma
+ */
+
+import clsx from 'clsx';
+import React, { HTMLAttributes, PropsWithChildren } from 'react';
+import { Heading } from './Heading';
+import { Link } from './Link';
+
+interface CardListItemProps extends Omit<HTMLAttributes<HTMLLIElement>, 'children'> {
+  headingLevel: number;
+  title?: string;
+  preHeading?: string;
+  children?: any;
+  imageHeight?: number;
+  imageWidth?: number;
+  imageSrc?: string;
+  imageAlt?: string;
+  href?: string;
+  cardRole?: 'article' | string;
+}
+
+export const CardListItem = ({
+  headingLevel,
+  children,
+  title,
+  preHeading,
+  imageHeight,
+  imageWidth,
+  imageSrc,
+  imageAlt,
+  href,
+  cardRole,
+  ...props
+}: PropsWithChildren<CardListItemProps>) => {
+  const linkRef = React.useRef<HTMLAnchorElement>(null);
+
+  let headingContent: any = title;
+  if (typeof href === 'string') {
+    headingContent = <Link href={href}>{title}</Link>;
```

Comment:
We moeten hier een property toevoegen om de Next.js, Gatsby of React-Router-link te kunnen gebruiken. We hebben deze aanpassing gedaan voor de IndexCharNav-component.

https://github.com/nl-design-system/utrecht/blob/main/packages/component-library-react/src/IndexCharNav.tsx#L59